### PR TITLE
Kraken: Dijkstra multi start

### PR DESCRIPTION
**Pull request by benoit-bst**
JIRA : https://jira.kisio.org/browse/NAVP-900

### CODE REVIEW

File: source/georef/street_network.cpp [hunk 1]
```diff
@@ -266,20 +266,15 @@ void PathFinder::start_distance_dijkstra(const navitia::time_duration& radius) {
     // We start dijkstra from source and target nodes
     try {
 #ifndef _DEBUG_DIJKSTRA_QUANTUM_
```
Comment (nonifier) [inline source/georef/street_network.cpp:268]: ! DIJKSTRA QUANTUM !

**Review by nonifier — APPROVED** [review]

File: source/georef/street_network.cpp (hunk 1 above)
Comment (benoit-bst) [reply source/georef/street_network.cpp:268]: hahaha

**Review by xlqian — APPROVED** [review]

File: source/georef/street_network.h [hunk 2]
```diff
@@ -173,9 +173,11 @@ struct PathFinder {
      * Warning, it modifies the distances and the predecessors
      **/
     template<class Visitor>
-    void dijkstra(vertex_t start, Visitor visitor) {
+    void dijkstra(vertex_t source, vertex_t target, Visitor visitor) {
         // Note: the predecessors have been updated in init
 
+        std::vector<georef::vertex_t> vertex {source, target};
```
Comment (TeXitoi) [inline source/georef/street_network.h]: This allocation can be skipped.

**Review by TeXitoi — APPROVED** [review]

File: source/georef/street_network.h (hunk 2 above)
Comment (benoit-bst) [reply source/georef/street_network.h]: :white_check_mark:

**Review by TeXitoi — APPROVED** [review]

**Pull request merged**